src/mixins.ts

```diff
@@ -38,6 +38,7 @@ export default {
       return shorten(str);
     },
     _ipfsUrl(ipfsHash: string): string {
+      if (ipfsHash === undefined) return '';
```

Comment:
I think it's something that should be solved before calling _ipfsUrl, ipfsHash should never get undefined.